addons/website_sale/models/product_product.py

```diff
@@ -14,7 +14,7 @@ class Product(models.Model):
 
     website_url = fields.Char('Website URL', compute='_compute_product_website_url', help='The full URL to access the document through the website.')
 
-    base_unit_count = fields.Float('Base Unit Count', required=True, default=1, help="Display base unit price on your eCommerce pages. Set to 0 to hide it for this product.")
+    base_unit_count = fields.Float('Base Unit Count', required=False, default=1, help="Display base unit price on your eCommerce pages. Set to 0 to hide it for this product.")
```

Comment:
Why this change?

Reply:
It was for a test failing about null constraint on the base_unit_count , i will try to run it on runbot to see if the change is very usefull by putting True instead of False 

> Roughly read through the code, the groups should not be removed in the upgrade script but rather renamed to be owned by `website_sale` instead, as such clients won't lose their settings.
And about that, quick question on what you advice, maybe using move_to_field in account/saas~15.5.1.2 ?

Reply:
> Roughly read through the code, the groups should not be removed in the upgrade script but rather renamed to be owned by `website_sale` instead, as such clients won't lose their settings.
And about that, quick question on what you advice, maybe using move_to_field in account/saas~15.5.1.2 ?

Reply:
I revert this change, wasn't useful, thx !

Reply:
> > Roughly read through the code, the groups should not be removed in the upgrade script but rather renamed to be owned by `website_sale` instead, as such clients won't lose their settings.
> 
> And about that, quick question on what you advice, maybe using move_to_field in account/saas~15.5.1.2 ?

You should use `rename_xmlid` in this case to set its new module. Upgrade will confirm if it's the right way to do :)